src/com/moclojer/config.clj

```diff
@@ -28,6 +28,10 @@
             :alias   :m}
    :version {:desc   "Show version."
              :alias  :v}
+   :format {:ref     "<format>"
```

Comment:
~~`--format json`~~ **is bad**

I don't like the name ﻿`--format`, I believe it is better to make explicit what this 'formatting' is
`﻿--log-format` I believe is better because it is explicit and not implicit

Reply:
what about the short form? `-l`?

Reply:
I like the short suggestion